### ISSUE TITLE
Getting "Google couldn't verify this account belongs to you." error after following gam-shortn URL

### ISSUE THREAD

**Issue by clader** [jay0lee/GAM#1157]
I'm trying to do a clean install of gam on a system using the latest (5.04) version.
After running the curl command, gam installs, and then I am prompted if I want to continue to create the Google Project, which I confirm I want to.

I then provide the full email of my Gsuite Superadmin user account, and Gam provides me with a:
https://gam-shortn.appspot.com/XXXXXX type url which I copy and paste into a browser window where I am already logged into Gsuite Admin as that user (and not logged to any other google account).
I then get a "choose account from xxx domain" screen where it shows my currently logged in (superadmin) user in the list, which I click to select.

Then I am immediately shown an error screen that says "Couldn't Sign you in. Google Couldn't verify this account belongs to you. Try again later or use Account Recovery to for Help"

If I click "recover account" it re-prompts me for that users password, which I enter, and it accepts as a valid password, and then just brings me to my "google account" screen (ignoring the gam-shortn url I was originally trying to browse to).

If I then try to browse to the gam-shortn url, the same process above happens.

Not really sure what is going on here as I am 100% logged as the same user whose email I provided to gam on the command line setup, and that user is a superadmin in the Gsuite domain. Installed Gam many times before for this user/domain with no issues, but this is the first time installing 5.04

**Comment by taers232c** [comment]
Do you already have a clean install of Gam on any computer? If so, you don't want/need a new project. Just copy client_secrets.json, oauth2service.json and oauth2.txt from the existing computer to the new computer.

Ross

**Comment by clader** [comment]
I do have an existing copy of Gam running for this domain, but it's on a machine at work I don't currently have access to, hence trying to get it setup again on my laptop ☹️ 

I actually have an existing gam instances setup on this laptop, but connected to a different Gsuite domain, and it was running a 4.xx version.  I was able to do the curl upgrade command on that instance and it upgraded to 5.04 with no issues.

It's just that doing a new project setup under 5.04 seems to have this account verification issue/error.

**Comment by jay0lee** [comment]
Can you post a screenshot of the error message?

What type of G Suite domain is this? If you go under the Billing section of the admin console, what type of licenses do you see?

**Comment by clader** [comment]
Sure thing here is the screenshot including the url, (but masked out the domain)
![Screen Shot 2020-04-10 at 7 23 56 PM](https://user-images.githubusercontent.com/9959496/79033196-0c804b00-7b61-11ea-827b-51181b3c0758.png)

**Comment by jay0lee** [comment]
What type of G Suite domain is this? If you go under the Billing section of the admin console, what type of licenses do you see?

**Comment by clader** [comment]
It's a Gsuite Enterprise account.

**Comment by clader** [comment]
I figured it, out and it was not related to GAM.  Our security team configured MFA/Security Key enforcement on our domain, so even though I could access the account and admin.google.com, i was being blocked from accessing the developer console until I added a security key to my Gsuite admin account.  

Marking this closed, and sorry for the false alarm. It was just that the error was not descriptive in any way of the root cause.
thx
